src/Microsoft.Graph.Cli.Core/Authentication/AuthenticationServiceFactory.cs

```diff
@@ -64,6 +68,8 @@ public virtual async Task<TokenCredential> GetTokenCredentialAsync(Authenticatio
                 return GetClientCertificateCredential(tenantId, clientId, certificateName, certificateThumbPrint);
             case AuthenticationStrategy.Environment:
                 return new EnvironmentCredential(tenantId, clientId);
+            case AuthenticationStrategy.ManagedIdentity:
+                return new ManagedIdentityCredential(clientId);
```

Comment:
Does managed identity not use a tenant id?

Reply:
I've just read [this](https://learn.microsoft.com/en-us/dotnet/api/azure.identity.managedidentitycredential?view=azure-dotnet) and seen that it does not.